projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/TestLifetimeAnalysisGCC.java

```diff
@@ -68,14 +80,14 @@
 import com.oracle.truffle.llvm.parser.impl.lifetime.LLVMLifeTimeAnalysisVisitor;
 import com.oracle.truffle.llvm.runtime.LLVMLogger;
 import com.oracle.truffle.llvm.runtime.options.LLVMBaseOptionFacade;
-import com.oracle.truffle.llvm.test.FunctionVisitorIterator.LLVMFunctionVisitor;
 
 @RunWith(Parameterized.class)
 public class TestLifetimeAnalysisGCC extends TestSuiteBase {
 
     // TODO in the long term we should replace this
     private static final String[] excludedFiles = new String[]{
-                    "projects/com.oracle.truffle.llvm.test/suites/gcc/gcc-5.2.0/gcc/testsuite/gcc.dg/pr43419.c"
+                    "projects/com.oracle.truffle.llvm.test/suites/gcc/gcc-5.2.0/gcc/testsuite/gcc.dg/pr43419.c",
```

Comment:
Could you please shortly comment why you excluded the test cases as part of the PR description?